tests/test_pipelines.py

```diff
@@ -981,14 +983,14 @@ def test_score_sde_ve_pipeline(self):
         sde_ve.to(torch_device)
         sde_ve.set_progress_bar_config(disable=None)
 
-        torch.manual_seed(0)
-        image = sde_ve(num_inference_steps=300, output_type="numpy").images
+        generator = torch.manual_seed(0)
+        image = sde_ve(num_inference_steps=300, output_type="numpy", generator=generator).images
```

Comment:
Can we reduce this here to less inference steps ? 300 takes too long and slows us down significantly when running the whole test suite